mistralrs-core/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![deny(clippy::cast_possible_truncation, clippy::cast_precision_loss)]
-
+#![feature(is_none_or)]
```

Comment:
Can you please remove this addition? It requires the nightly compiler, not sure if this was a leftover.

Reply:
> Can you please remove this addition? It requires the nightly compiler, not sure if this was a leftover.

Yes, I will remove this. I use nightly compiler because program compiled by the stable one has conflicts with lldb debugger on my Mac (unable to show concrete values for vec variables).